subgraph-js/src/buildSubgraphSchema.ts

```diff
@@ -9,13 +9,14 @@ import {
   specifiedDirectives,
 } from 'graphql';
 import {
-  buildSchemaFromSDL,
-  transformSchema,
   GraphQLSchemaModule,
-  modulesFromSDL,
-  addResolversToSchema,
   GraphQLResolverMap,
-} from 'apollo-graphql';
+  addResolversToSchema,
+  modulesFromSDL,
+  transformSchema,
```

Comment:
It seems (at least this fn) is only used by the subgraph package, can it live there instead and not in the gateway? Also, this package doesn't define a dependency on gateway (and I would prefer if it continued not to). Are we able to keep it that way?

Reply:
I think there are three options, I'm happy to do any of them, just let me know which you prefer.
1. Make a new package for the common code and put it there (or in internals-js)
2. Duplicate the code in both gateway-js and subgraph-js. The only function used by both is `addResolversToSchema` + the interfaces
3. Make gateway-js dependent on subgraphs-js.

Reply:
👍 for option 2 (not too worried about duplication in the short term, should have a solution for this very soon). Given that, I'd rather avoid adding dependencies on each other unless it really makes sense to.